notebooks/124-hugging-face-hub/124-hugging-face-hub.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Stable Diffusion XL, Riffusion, Dolly v2, llm chatbot also should be here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>added</p>